RibonDAO/core-api
# (TECH-114): webhook for stripe

# Description

- Webhook for refund events at stripe

# Necessary changes

- Activate stripe webhooks


## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Refund a payment on stripe and see if the person payment status was updated
